### ISSUE TITLE
feat(matDatepicker): How to do month range picking with datepicker

### ISSUE THREAD

**Issue by mithun2003** [angular/components#29182]
### Feature Description

I want to pick two month range for filter but by default it give date also so how to select month range only

**Comment by forgantini** [comment]
i'm interested into too

**Comment by amysorto** [comment]
Do you mean you want a daterange picker that shows two months side by side? Something along the lines like this?
![Image](https://github.com/angular/components/assets/8575252/75918693-9f0a-4b44-8767-b57a53d887b0)

Or a month range picker (ex. April to September). If so it seems like thats probably too small of a use case to add to this library and something that might be better as a custom component in your app.

**Comment by forgantini** [comment]
no simply a month range picker like the day range is (something like that for example: https://stackblitz.com/edit/angular-kbsxte?file=src%2Fapp%2Fmonth-picker%2Fmonth-picker.component.ts ) ... i don't know if exists some configuration to make the rangedatepicker act in this way (public or private) but could be useful

**Comment by mithun2003** [comment]
> no simply a month range picker like the day range is (something like that for example: https://stackblitz.com/edit/angular-kbsxte?file=src%2Fapp%2Fmonth-picker%2Fmonth-picker.component.ts ) ... i don't know if exists some configuration to make the rangedatepicker act in this way (public or private) but could be useful 

Yeah I want a month picker like this. I think we can't create it using matdaterangepicker

**Comment by mmalerba** [comment]
There's an example in the docs of how to emulate this behavior: https://material.angular.dev/components/datepicker/overview#datepicker-views-selection
